package.json

```diff
@@ -17,5 +17,8 @@
   "author": {
     "name": "Netherlands3D",
     "url": "https://netherlands3d.eu"
+  },
+  "dependencies": {
+    "eu.netherlands3d.filebrowser": "2.1.0"
```

Comment:
Zie ik het goed dat de assembly def refeert naar de sun package maar dat deze niet als dependency hier staat? Oj. Ik weet dat het niet uit jouw changes komt, maar zou je die dependency willen toevoegen omdat het een directe dependency is?